noetic/distribution.yaml

```diff
@@ -6602,6 +6602,22 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/robotis_manipulator.git
       version: noetic-devel
     status: maintained
+  rom_motor_msgs:
+    doc:
+      type: git
+      url: https://github.com/GreenGhostMan/rom_motor_msgs-doc.git
+      version: main
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/GreenGhostMan/rom_motor_msgs-release.git
+      version: 0.0.2-1
+    source:
+      test_pull_requests: true
```

Comment:
@GreenGhostMan have you [configured your repository](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to allow this?

Reply:
@hidmic yes